### ISSUE TITLE
set-sequence-music command

### ISSUE THREAD

**Issue by bookdude13** [teslaworksumn/proton-cli#2]
Implement command to change the music source of a sequence. Automatically resections afterwards based on the new music file's duration.
`set-sequence-music <admin-key> <name> <music-path>`

**Comment by computergeek125** [comment]
This one especially should need an undo button or a warning message.  If you automatically resize a song to longer than the original array, that's fine, but making it shorter is problematic.  What if a sequencer had data at the end of the song?  We don't just want to kill that.

Idea: Only pop the warning message if data exists in the section of the song that's going to be deleted.

**Comment by bookdude13** [comment]
Good point. The advantage of using Git to manage the data is that we _could_ go back and recover the data, but it would be a manual workaround. Warnings are good.
